.github/workflows/ci.yml

```
@@ -2,59 +2,135 @@
+      fail-fast: false
+      matrix:
+        svc: [jira-fetch-service, reporting-service, chart-service, email-service, excel-service, gateway-service, discovery-service]
+    steps:
+      - name: Trivy scan ${{ matrix.svc }}
+        uses: aquasecurity/trivy-action@0.24.0
+        with:
+          image-ref: ghcr.io/${{ github.repository_owner }}/${{ matrix.svc }}:${{ github.sha }}
+          format: table
+          vuln-type: os,library
+          severity: CRITICAL,HIGH
+          exit-code: '1'
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)